contracts/Cartographer.sol

```diff
@@ -901,6 +914,11 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
                 false
             );
 
+        // native farm bonus handling
+        if (isNativeFarmToken[_token]) {
+            nativeFarmTokenLastDepositTimestamp[msg.sender][_token] == 0;
```

Comment:
This should set it to the current `block.timestamp` so that it can start ticking up 7 days from this point